myuw/static/vue/components/_common/course/course-eval.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div v-if="evalData && evalData.length > 0">
+    <h5>Course Evaluations</h5>
+    <p>
+      <span>Evaluations close {{ toFriendlyDate(eval.close_date) }}</span> (at 11:59PM), and only take a few minutes to complete.
+    </p>
+
+    <div v-for="(eval, index) in evalData" :key="index">
+      <template v-if="eval.is_multi_instr">
+        <!-- evaluation is on the course -->
+        <a href="`${eval.url}`" target="_blank">
+          {{ section.curriculum_abbr }} {{ section.course_number }} {{ section.section_id }} Evaluation</span>
+        </a>
+        <ul>
+          <li v-for="(instructor, index) in eval.instructors" :key="index">
+            <span>
+              {{ instructor.instructor_name }}
+            </span>
+            <span v-if="instructor.instructor_title.length > 0">
+              {{ instructor.instructor_title }}
+            </span>
+          </li>
+        </ul>
+      </template>
+      <template v-else>
+        <ul>
+          <li v-for="(instructor, index) in eval.instructors" :key="index">
+            <a href="`${eval.url}`" target="_blank">
```

Comment:
this should be `:href="eval.url"`. The `:` tells vue to read the value as a js variable